### PR TITLE
Adding builtin GraphQL Mediator to the Synapse

### PR DESCRIPTION
**Pull request by ajanthasingam**
### Purpose
Implement builtin GraphQL Mediator

### CODE REVIEW

File: modules/core/src/main/java/org/apache/synapse/mediators/builtin/GraphQLMediator.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    }
+
+    private GraphQLSchema buildSchema(String sdl)
+            throws  ClassNotFoundException, IllegalAccessException, InstantiationException {
+
+        TypeDefinitionRegistry typeRegistry = new SchemaParser().parse(sdl);
+
+        Class clazz = this.getClass().getClassLoader().loadClass(runtimeWiringClassName.trim());
+        RuntimewiringInterface datafetcher = (RuntimewiringInterface) clazz.newInstance();
+
```
Comment (sameeragunarathne) [inline modules/core/src/main/java/org/apache/synapse/mediators/builtin/GraphQLMediator.java]: remove extra line

File: modules/core/src/main/java/org/apache/synapse/mediators/builtin/GraphQLMediator.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        RuntimeWiring runtimeWiring  = datafetcher.buildWiring();
+
```
Comment (sameeragunarathne) [inline modules/core/src/main/java/org/apache/synapse/mediators/builtin/GraphQLMediator.java]: remove extra line

File: modules/core/src/main/java/org/apache/synapse/mediators/builtin/GraphQLMediator.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+    private void init() {
+        try {
+            URL url = new File(schemaPath).toURI().toURL();
+            String sdl = Resources.toString(url, Charsets.UTF_8);
+            GraphQLSchema graphQLSchema = buildSchema(sdl);
+            this.graphql = GraphQL.newGraphQL(graphQLSchema).build();
+        } catch (Exception e) {
+            e.printStackTrace();
```
Comment (menuka94) [inline modules/core/src/main/java/org/apache/synapse/mediators/builtin/GraphQLMediator.java]: Let's use a logger here.

File: modules/core/src/main/java/org/apache/synapse/config/xml/JWTMediatorFactory.java [hunk 4]
```diff
@@ -0,0 +1,109 @@
+package org.apache.synapse.config.xml;
```
Comment (Nirothipan) [inline modules/core/src/main/java/org/apache/synapse/config/xml/JWTMediatorFactory.java]: Please include the license header

File: modules/core/src/main/java/org/apache/synapse/mediators/builtin/JWTMediator.java [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+
+        byte[] rsaKey =  Base64.getDecoder().decode(publicKey);
+
+        X509EncodedKeySpec spec =  new X509EncodedKeySpec(rsaKey);
+        rsaPublicKey = (RSAPublicKey) kFactory.generatePublic(spec);
+
+
+    }
+
+
```
Comment (Nirothipan) [inline modules/core/src/main/java/org/apache/synapse/mediators/builtin/JWTMediator.java]: Please remove these empty lines

**Comment by isudana** [comment]
Please re-open after making requested changes

**Pull request closed without merging**